src/Console/Command/DescribeCommand.php

```diff
@@ -183,18 +184,19 @@ private function describeSet(InputInterface $input, OutputInterface $output, $na
 
         $fixers = $this->getFixers();
 
-        $output->writeln(sprintf('<info>Description of %s set.</info>', $name));
+        $output->writeln(sprintf('<info>Description of</info> %s <info>set.</info>', $name));
         $output->writeln('');
 
         $help = '';
 
         foreach ($rules as $rule => $config) {
+            /** @var FixerDefinitionInterface $definition */
+            $definition = $fixers[$rule]->getDefinition();
             $help .= sprintf(
-                " * <info>%s</info>%s\n   | %s\n%s\n",
+                " * <info>%s</info>%s%s\n",
                 $rule,
-                $fixers[$rule]->isRisky() ? ' <error>[risky]</error>' : '',
-                $fixers[$rule]->getDescription(),
-                true !== $config ? sprintf("   <comment>| Configuration: %s</comment>\n", $this->arrayToText($config)) : ''
```

Comment:
why dropping a configuration here ? sets are not always following default config.